docs/user-guide/value_federates.md

```diff
@@ -64,7 +64,7 @@ Though contained here in this section on value federates, the options below are
      ],
      "subscriptions" : [
           {
-               "required": true,
```

Comment:
I am thinking we might want to allow required for the time being since I know of a number of people using this option yet?

Reply:
I'm OK with that. My preference was for more verbose option names to remove confusion but consider the popularity of this one. We can keep it around for now. Do we want to add a deprecation warning?

Reply:
I added the required string back in so that should still work, but we will mark it as deprecated at some point in the future